npm/css-src/sass/_drag-and-drop.scss

```diff
@@ -64,3 +64,12 @@
 .error-messages {
   margin-bottom: 0;
 }
+
+.success-message-flexbox-container {
+  align-items: center;
+  display: flex;
+}
+
+.success-message-flexbox-item-2 {
```

Comment:
From my other comment about removing the `success-message-flexbox-item-1` class. If you do that, can this be renamed to `success-message-flexbox-item` ?